src/metabase/channel/render/card.clj

```diff
@@ -200,13 +202,14 @@
                         ;; Provide a horizontal scrollbar for tables that overflow container width.
                         ;; Surrounding <p> element prevents buggy behavior when dragging scrollbar.
                         [:div
-                         [:a {:href        attachment-href
-                              :target      "_blank"
-                              :rel         "noopener noreferrer"
-                              :style       (style/style
-                                            (style/section-style)
-                                            {:display         :block
-                                             :text-decoration :none})}
+                         [:a (cond-> {:target      "_blank"
+                                      :rel         "noopener noreferrer"
+                                      :style       (style/style
+                                                    (style/section-style)
+                                                    {:display         :block
+                                                     :text-decoration :none})}
+                               (not (:channel.render/disable-links? options))
+                               (assoc :href attachment-href))
```

Comment:
not including links on the whole card.

<img width="699" height="428" alt="Image" src="https://github.com/user-attachments/assets/3c67b062-ab16-4671-99fb-60cec9e3c2f8" />


P.S. I'm not sure why we need both links in the title and the card tbh, as the card already covers the title.